journal/modernized/15-chapter-13.adoc

```diff
@@ -248,8 +254,10 @@ and answer your supplications, tears, sighs and groans.
 God Almighty be with you, and fill you with the fillings of his life and heavenly wisdom,
 is the breathing and travail of the soul of your bowed-spirited friend and brother,
 
+[.signed-section-signature]
 Charles Marshall.
 
+[.postscript]
```

Comment:
This postscript looked a little funny too.

Reply:
yeah, I'd mark this one with an [.offset] too. 👍🏻